diagnostic.md

```diff
@@ -51,19 +51,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+Absolute Path
  <!-- Answer Ends Here -->
 
  2. Suppose that we're working on a project, and we want to use a font that's being hosted on Google at `https://fonts.googleapis.com/css?family=Open+Sans`. Is that an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+Absolute
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Not sure I understand this question. inside what project? /Users/blah_blah/Desktop? inside the diagnostic? if diagnostic then relative because we are in it and don't need to be specific.
```

Comment:
This question is ambiguous and I put in an issue with it.  The project can be anywhere and both absolute paths and relative paths can be used.  Relative paths are usually used since they are (usually) less verbose, so long as the project and all of its referenced files aren't moved.